src/status.js

```diff
@@ -1,6 +1,13 @@
-export default function updateStatus(event) {
+import { getFromStorage, saveToStorage } from './storage.js';
+
+export function updateStatus(event) {
```

Comment:
Kindly do the same here too, introduce a bug in this function and run your test, check if your test will catch the bug.

It doesn't, kindly check my comment and use the hint.